usaspending_api/awards/award_type.py

```diff
@@ -0,0 +1,53 @@
+from django.db.models import F, Sum
```

Comment:
Let's create a `views_v2` directory and place these views in there. We can also consider grouping the views into something like `award_spending.py` since both endpoints fall under that higher level category.